src/v3/test/integration/authenticator-enroll-data-phone.test.tsx

```diff
@@ -107,7 +92,7 @@ describe('authenticator-enroll-data-phone', () => {
 
     const submitButton = await findByText('Receive a code via voice call');
     const phoneNumberEle = await findByLabelText('Phone number') as HTMLInputElement;
-    const extensionEle = await findByLabelText('Extension') as HTMLInputElement;
+    const extensionEle = await findByLabelText('ExtensionOptional') as HTMLInputElement;
```

Comment:
Since we made the field optional, MUI adds the word "Optional" into the label (right aligned).